src/aat/resources/features/F-026/S-076.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "must return appropriate negative response when request does not provide valid authentication credentials",
```

Comment:
Title doesn't match with the scenario description in the feature file.

Reply:
Updated:   "title": "must return 412 when the case type is not present in Definition store workbasket input fields",